### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by jimmyli97**

### CODE REVIEW

File: basex/src/main/java/com/formulasearchengine/mathosphere/basex/types/Formula.java [hunk 1]
```diff
@@ -19,24 +19,35 @@
  * Created by jjl4 on 6/24/15.
  */
 @XStreamAlias("formula")
-@XStreamConverter(Formula.FormulaConverter.class)
+//@XStreamConverter(Formula.FormulaConverter.class)
```
Comment (physikerwelt) [inline basex/src/main/java/com/formulasearchengine/mathosphere/basex/types/Formula.java]: remove that line!

**Comment by physikerwelt** [comment]
:+1: otherwise it looks good

**Comment by jimmyli97** [comment]
fixes benchmark in #73

**Comment by physikerwelt** [comment]
feel free to merge

**Pull request merged**